airbyte-integrations/connector-templates/source-java-jdbc/integration_tests/acceptance.py

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
```

Comment:
I think you can copy all the content of this folder `airbyte-integrations/connector-templates/source-python/integration_tests`

Reply:
I worry that we can make this generator worth it if we add things we don't use. At least until we enabled DISCOVER and READ tests for the existing source and specify what we need and why.
I remember the expectation of enabling SPEC and what we have. Here we might have even more changes required besides just turning it on.

Reply:
> Here we might have even more changes required besides just turning it on.

I don't get which change you mean.
IMO these files are only examples that could be used in the SAT context.
Developers are free to remove these if it's not used. 
When a new connector gets generated it's not expected they directly pass all SATs. So showing devs what can be tested remains valuable. We have no clue that a new JDBC connector can't pass tests other than SPEC, so let's offer them the opportunity to do so. Wdyt?

Reply:
I've managed to integrate the BigQuery source with the SAT in the PR. As a result, I got a list of files and required content to add here.